album.go

```diff
@@ -136,13 +137,13 @@ func toStringSlice(ids []ID) []string {
 // Spotify IDs.  It supports up to 20 IDs in a single call.  Albums are returned
 // in the order requested.  If an album is not found, that position in the
 // result slice will be nil.
-func (c *Client) GetAlbums(ids ...ID) ([]*FullAlbum, error) {
-	return c.GetAlbumsOpt(nil, ids...)
+func (c *Client) GetAlbums(ctx context.Context, ids ...ID) ([]*FullAlbum, error) {
+	return c.GetAlbumsOpt(ctx, nil, ids...)
 }
 
 // GetAlbumsOpt is like GetAlbums but it accepts an additional country option for track relinking
 // Doc API: https://developer.spotify.com/documentation/web-api/reference/albums/get-several-albums/
-func (c *Client) GetAlbumsOpt(opt *Options, ids ...ID) ([]*FullAlbum, error) {
+func (c *Client) GetAlbumsOpt(ctx context.Context, opt *Options, ids ...ID) ([]*FullAlbum, error) {
```

Comment:
Is the plan to replace `*Options` with the functional options pattern everywhere?

Reply:
Yup, that would be my plan if you are happy with the options as it is currently looking? I didn't want to go ahead and refactor everywhere until you seemed happy with it.